packages/runtime/src/parser/parser.ts

```diff
@@ -894,7 +885,14 @@ function _assertMaxLen(len: number, mode: ExecutionMode): void {
 export function parser(program: string, mode: ExecutionMode, interpreter: Interpreter): Op[] {
 	const opCodeList: Op[] = [];
 	let counter = 0;
-
+	let lsigProgramArgsSize = Buffer.from(program, "base64").length;
```

Comment:
the size doesn't refer to compiled version?

Reply:
No, we don't need that.
I have checked how they have verified this case over here: Please see here: https://github.com/algorand/go-algorand-sdk/blob/13da8508530b7dc2e9ae62766cccd0baaff0f4bd/logic/logic.go#L83

They are using bytecode of program.

Reply:
The argument passed there is type `byte` and we use string. Lets ask on Algorand discord!